src/utils/logger.ts

```diff
@@ -1,31 +1,57 @@
 import { createLogger, format, transports } from "winston";
 import { env } from "./env";
 
+type LogLevels = typeof env.LOG_LEVEL;
+
+// Define custom log levels that strictly match the log levels in the env file
+const customLevels: {
+  levels: { [K in LogLevels]: number };
+  colors: { [K in LogLevels]: string };
+} = {
+  levels: {
+    fatal: 0,
+    error: 1,
+    warn: 2,
+    info: 3,
+    debug: 4,
+    trace: 5,
+  },
+  colors: {
+    fatal: "red",
+    error: "red",
+    warn: "yellow",
+    info: "green",
+    debug: "blue",
+    trace: "gray",
+  },
+};
+
 // Custom filter for stdout transport
-const filterOnlyInfoAndWarn = format((info) => {
-  if (info.level === "error") {
-    return false; // Exclude 'error' level logs
+const filterNonErrors = format((info) => {
+  if (info.level !== "error" && info.level !== "fatal") {
+    return info; // only include non-error logs
   }
-  return info;
+  return false;
```

Comment:
This logic is correct but seems like it can be more readable:

Instead of:
```
if not error and not fatal
  return info
else
  return false
```

This seems like the same but more readable:
```
if error or fatal
  return false
else
  return info
```